Content/default/Build.fs

```diff
@@ -78,6 +74,10 @@ Target.create "Format" (fun _ -> run dotnet [ "fantomas"; "." ] ".")
 open Fake.Core.TargetOperators
 
 let dependencies = [
+    "Build" ==> "RunTestsHeadless"
+    "Build" ==> "WatchRunTests"
+    "Build" ==> "Run"
```

Comment:
Could you merge this with the Run pipeline below?

Reply:
```
let dependencies = [
    "Clean" ==> "RestoreClientDependencies" ==> "Bundle" ==> "Azure"
    "Clean" ==> "RestoreClientDependencies" ==> "Build" ==> "Run"

    "RestoreClientDependencies" ==> "Build" ==> "RunTestsHeadless"
    "RestoreClientDependencies" ==> "Build" ==> "WatchRunTests"
]
```
Like so?